presto-main/src/test/java/com/facebook/presto/type/TestDoubleOperators.java

```diff
@@ -177,6 +178,24 @@ public void testCastToBigint()
     {
         assertFunction("cast(37.7E0 as bigint)", BIGINT, 38L);
         assertFunction("cast(17.1E0 as bigint)", BIGINT, 17L);
+        assertFunction("cast(9.2E18 as bigint)", BIGINT, 9200000000000000000L);
+        assertFunction("cast(-9.2E18 as bigint)", BIGINT, -9200000000000000000L);
+        assertFunction("cast(2.21E9 as bigint)", BIGINT, 2210000000L);
+        assertFunction("cast(-2.21E9 as bigint)", BIGINT, -2210000000L);
+
+        assertFunction("cast(" + Math.nextDown(0x1.0p63) + " as bigint)", BIGINT, (long) Math.nextDown(0x1.0p63));
+        assertFunction("cast(" + Math.nextUp(-0x1.0p63) + " as bigint)", BIGINT, (long) Math.nextUp(-0x1.0p63));
+        assertFunction("cast(" + -0x1.0p63 + " as bigint)", BIGINT, (long) -0x1.0p63);
+        assertInvalidFunction("cast(" + Math.nextUp(0x1.0p63) + " as bigint)", INVALID_CAST_ARGUMENT);
+        assertInvalidFunction("cast(" + Math.nextDown(-0x1.0p63) + " as bigint)", INVALID_CAST_ARGUMENT);
+        assertInvalidFunction("cast(" + 0x1.0p63 + " as bigint)", INVALID_CAST_ARGUMENT);
```

Comment:
Please reorder these 6 lines:

* Have the 3 lines for 0x1.0p63 together. (ditto for -0x1.0p63)
* Within each 3 lines, please put the line without nextUp/nextDown in the middle.
* Please use the same order for 0x1.0p63 lines and -0x1.0p63 lines.